workspace/apps/pidp/src/app/features/shell/pages/support-error/support-error.page.scss

```diff
@@ -0,0 +1,15 @@
+@import 'palette';
+
+.error-page {
+  display: flex;
+  flex-direction: column;
+  color: #fff;
+  background-color: theme-palette(blue);
+  height: 100vh;
+  padding: 30px;
+}
+
+button {
+  border: 1px solid #fff;
+  max-width: 300px;
```

Comment:
You took this a bit too far.  You can wrap the button in a `<span>` so it maintains it's own width based on inner content, but removing `<div>` off the `<h1>` is perfect since that was diviitus.